doc/guides/writing-tests.md

```diff
@@ -18,6 +18,14 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Test directory structure
+
+See the test [directory structure overview][] for an outline of existing test types and their locations.
+When deciding on whether to expand an existing test file or create a new one.
```

Comment:
The punctuation at the end should be `,`

Reply:
changes made :)